bin/linguist

```diff
@@ -17,20 +17,48 @@ if path == "--breakdown"
 elsif path == "--json"
   path = Dir.pwd
   json_breakdown = true
+elsif path == "-b"
+  path = Dir.pwd
+  bytes = true
 end
 
 ARGV.shift
 breakdown = true if ARGV[0] == "--breakdown"
 json_breakdown = true if ARGV[0] == "--json"
+bytes = true if ARGV[0] == "-b"
+
+ARGV.shift
+bytes = true if ARGV[0] == "-b"
 
 if File.directory?(path)
   rugged = Rugged::Repository.new(path)
   repo = Linguist::Repository.new(rugged, rugged.head.target_id)
   if !json_breakdown
     repo.languages.sort_by { |_, size| size }.reverse.each do |language, size|
-      percentage = ((size / repo.size.to_f) * 100)
-      percentage = sprintf '%.2f' % percentage
-      puts "%-7s %s" % ["#{percentage}%", language]
+      if !bytes
+        percentage = ((size / repo.size.to_f) * 100)
+        percentage = sprintf '%.2f' % percentage
+        puts "%7s %s" % ["#{percentage}%", language]
+      else
+        size = size.to_f
+        if size < 1024
+          unit = "B"
+        elsif size >=1024 && size < 1048575
+          unit = "K"
+          size /= 1024.0
+        elsif size >= 1048576 && size < 1073741824
+          unit = "M"
+          size /= 1048576.0
+        elsif size >= 1073741824 && size < 1099511627776
+          unit = "G"
+          size /= 1073741824.0
+        else
+          unit = "T"
+          size /= 1099511627776.0
+        end
+        ds = sprintf '%.2f' % size
+        puts "%8s %s" % ["#{ds}#{unit}", language]
+      end
```

Comment:
This should be in its own function. In Ruby, it can also be made shorter:
```ruby
GIGA = 1073741824.0
MEGA = 1048576.0
KILO = 1024.0
def readable_file_size(size)
  case
    when size < KILO : "%dB" % size
    when size < MEGA : "%.2fKB" % (size / KILO)
    when size < GIGA : "%.2fMB" % (size / MEGA)
    else "%.2fGB" % (size / GIGA)
  end
end
```